config/initializers/airbrake.rb

```diff
@@ -1,7 +1,12 @@
 # This file is overwritten on deploy
-#
+
 Airbrake.configure do |config|
+  config.api_key = ENV["ERRBIT_API_KEY"]
+  config.host    = ENV["ERRBIT_HOST"]
+  config.environment_name = ENV["ERRBIT_ENV"]
+  config.secure  = ENV["ERRBIT_API_KEY"].present?
+
   # Adding "production" to the development environments causes Airbrake not
   # to attempt to send notifications.
-  config.development_environments << "production"
+  config.development_environments << "production" unless ENV["ERRBIT_API_KEY"].present?
```

Comment:
Prior to https://github.com/airbrake/airbrake/pull/268 this was necessary to prevent the airbrake gem attempting to send error notices to airbrake when not configured (configuration was done by overwriting this file).  If this is using airbrake 3.1.17 or later, this should no longer be necessary.